src/components/search/Facets.js

```diff
@@ -42,36 +42,37 @@ LoadingOverlay.propTypes = undefined;
 
 const DateFacet = ({facetsToInclude}) => {
 
-  const searchState = useSelector(state => state.search);
+  const datePubmedModified = useSelector(state => state.search.datePubmedModified);
+  const datePubmedAdded = useSelector(state => state.search.datePubmedAdded);
   const dispatch = useDispatch();
 
   return (
     <div>
       <div key={facetsToInclude[0]} style={{textAlign: "left", paddingLeft: "2em"}}>
       <h5>{facetsToInclude[0]}</h5>
-        <DateRangePicker value={searchState.datePubmedModified} onChange= {(newDateRangeArr) => {
+        <DateRangePicker value={datePubmedModified} onChange= {(newDateRangeArr) => {
           if (newDateRangeArr === null) {
             dispatch(setDatePubmedModified(''));
             dispatch(setSearchResultsPage(0));
```

Comment:
the second and the third dispatch calls are always the same for both cases - they can be placed outside the if-else. Same thing for lines 70-71 and 74-75

Reply:
Moved these actions.